Add new known issue to "Installation known issues"

oafischer opened the pull request:
https://github.com/stolostron/backlog/issues/22382#issue-1229858793

@swopebe I copied the suggested info from @mikeshng ([link](https://github.com/stolostron/backlog/issues/22382#issuecomment-1121420180)) but made some style changes and formatted it based on the rest of the documentation. Please review to make sure the changes are OK.

oafischer committed:
Add new known issue

swopebe commented:
Sure thing, let's chat tomorrow, I will book some time. Looks pretty good, we can use it as a learning session.

swopebe committed:
Edit from meeting

swopebe commented on release_notes/known_issues.adoc:
```diff
@@ -57,6 +57,30 @@ If this happens, you can either find the specified section manually or complete
++
+----
+kubectl -n multicluster-engine get deploy cluster-curator-controller
+----
```
@oafischer do you mind checking on this? It seems two commands for the same thing, one is MCE. That means this may need to be an MCE known issue (new file in the folder, same set-up)

swopebe replied:
We need to make sure we specify why we have two commands here. Also @mikeshng the MCE doc is in a different folder and we are trying to make sure we discuss MCE in that folder only. It's tricky. I am thinking this needs to be a known issue in MCE, no?

oafischer replied:
@swopebe I checked in on this, I will clarify and make adjustments based on the new info I have that should hopefully resolve these questions.

swopebe replied:
We cleared this up in the meeting, for some reason the comments surfaced late.

oafischer reviewed (commented):
Thanks for the edits @swopebe !
@mikeshng confirmed that what we have now looks good.

oafischer committed:
Update known_issues.adoc

Appending the previously made edits after clarifying with @mikeshng and @swopebe

swopebe reviewed (approved):
looks great

oafischer committed:
Update known_issues.adoc

Switch order to make MCE command appear first (because MCE is mentioned first in step 1)
